app/views/message_threads/bulk/exports/edit.html.erb

```diff
@@ -10,9 +10,21 @@
           <h3 class="text-base font-semibold text-gray-900">Nastavenie exportu</h3>
         </div>
 
+        <% if @export.errors.any? %>
+          <div class="mt-4 rounded-md bg-red-50 p-4" data-controller="dismissible-alert" data-dismissible-alert-timeout-value="5000">
```

Comment:
Toto podla mna nechceme ako dismissible alert, normalne nech to tam zostane svietit pokym to pouzivatel nenapravi.